scoresheets/ObjectConfiguration.tex

```diff
@@ -0,0 +1,27 @@
+The maximum time for this test is 6 minutes.
+
+\begin{scorelist}
+	\scoreheading{Describing the objects} % 100
+	%\scoreitem[n]{50 \/ n}{Description touches upon each of the n objects}
+	\scoreitem[10]{5}{Description touches upon each of the n objects}
```

Comment:
I don't think is a good idea to have $n$ in any scoresheet. All teams should be able to score the same.

Reply:
the original idea was to have n objects, with each being worth (50/n) points, so that the "Describing the objects" in total is worth 50 points always, and n constant for the competition.

But perhaps having a constant value is better, so I changed n to 10.   Feel free to let me know which you like better